README.md

```diff
@@ -39,41 +39,63 @@ we would like things to flow.
+* OpenJDK 8
 * Network access to https://repository.sonatype.org/content/groups/sonatype-public-grid
 
 Also, there is a good amount of information available at [Bundle Development](https://help.sonatype.com/display/NXRM3/Bundle+Development#BundleDevelopment-BundleDevelopmentOverview)
 
-NOTE: Version 0.0.2+ of Conan is only compatible with Nexus Repo 3.11.0 or greater due to adding Tree View Filtering. If 
-you need to use this plugin with an older version, please use 0.0.1
-
 ### Building
 
 To build the project and generate the bundle use Maven
 
-    mvn clean package -PbuildKar
+    mvn clean install
 
 If everything checks out, the bundle for Conan should be available in the `target` folder
 
-#### Build with Docker
+## Using Conan With Nexus Repository Manager 3
 
-`docker build -t nexus-repository-conan:0.0.6 .`
+[We have detailed instructions on how to get started here!](https://help.sonatype.com/nxrm3master/formats/conan-repositories)
 
-#### Run as a Docker container
+## Compatibility with Nexus Repository Manager 3 Versions
 
-`docker run -d -p 8081:8081 --name nexus nexus-repository-conan:0.0.6` 
+The table below outlines what version of Nexus Repository Manager the plugin was built against
 
-For further information like how to persist volumes check out [the GitHub repo for our official image](https://github.com/sonatype/docker-nexus3).
+| Plugin Version    | Nexus Repository Manager Version |
+|-------------------|----------------------------------|
+| v0.0.1            | <3.11.0                          |
+| v0.0.2+           | 3.11.0+                          |
+| v1.0.0+ In product| 3.22.0+                          |
 
-The application will now be available from your browser at http://localhost:8081
 
-## Using Conan With Nexus Repository Manager 3
+## Installing the plugin
 
-[We have detailed instructions on how to get started here!](docs/CONAN_USER_DOCUMENTATION.md)
+In Nexus Repository Manager 3.22+ Conan proxy format is already included. So there is no need to install it.
+But if you want to reinstall the plugin with your improvements then following instructions will be useful.
+Note: Using an unofficial version of the plugin is not supported by the Sonatype Support team.
 
-## Installing the plugin
+### Permanent ReInstall
 
-There are a range of options for installing the Conan plugin. You'll need to build it first, and
+If you are trying to use the Conan plugin permanently, it likely makes more sense to do the following:
+
+* Copy the bundle into `<nexus_dir>/system/org/sonatype/nexus/plugins/nexus-repository-conan/1.0.0/nexus-repository-conan-1.0.0.jar`
+* If you are using OSS edition, make these mods in: `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-oss-feature/3.x.y/nexus-oss-feature-3.x.y-features.xml`
+* If you are using PRO edition, make these mods in: `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-pro-feature/3.x.y/nexus-pro-feature-3.x.y-features.xml`
+   ```
```

Comment:
Trivial: At first I felt there was a line break missing here.  Thinking about it more, I wonder if the 2 bullets preceeding are not actionable, they are guiding.  Wonder if it could be a bullet, "Modify xml to introduce the plugin" with subbullets of OSS/PRO "where".  I get it as written, but I also wonder if I'm just used to reading these and understand what you mean...2 cents.

Reply:
Changed e220634479ce99a37561b6c2b1bf7ee532887427 Is it better now?

Reply:
Looks great=)
Could put "In OSS..." and "In PRO..." but it's super trivial, just going to resolve.
Thanks!